Modules/Microsoft365DSC/DSCResources/MSFT_O365ExternalConnection/settings.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+    "resourceName": "O365ExternalConnection",
+    "description": "Configures external connectors in Microsoft 365.",
+    "roles": {},
+    "permissions": {
+        "graph": {
+            "delegated": {
+                "read": [],
+                "update": []
+            },
+            "application": {
+                "read": [
+                    {
+                        "name": "ExternalConnection.Read.All"
+                    }
```

Comment:
The module calls Get-MgApplication so this also needs Application.Read.All for both read and update.